samples/amber.cc

```diff
@@ -337,9 +337,21 @@ class SampleDelegate : public amber::Delegate {
       return r;
 #endif  // AMBER_ENABLE_LODEPNG
 
-    // TODO(asuonpaa): Try to load a binary format.
+    // Try to load a binary format.
+    auto data = ReadFile(path_ + file_name);
+    if (data.empty())
+      return amber::Result("Failed to load buffer data " + file_name);
 
-    return amber::Result("Failed to load buffer data " + file_name);
+    for (auto d : data) {
+      amber::Value v;
+      v.SetIntValue(d);
```

Comment:
Does this not need to match the DATA_TYPE of the buffer? If the buffer is all float data, why is this written as all Int's?

Reply:
Data is always loaded as byte format. There's no point converting the data here and then converting it back to bytes when loading the data into a buffer.